demo-app/src/main/java/cash/z/ecc/android/sdk/demoapp/demos/getaddress/GetAddressFragment.kt

```diff
@@ -30,23 +30,25 @@ class GetAddressFragment : BaseDemoFragment<FragmentGetAddressBinding>() {
             viewLifecycleOwner.repeatOnLifecycle(Lifecycle.State.STARTED) {
                 launch {
                     sharedViewModel.synchronizerFlow.filterNotNull().collect { synchronizer ->
+                        val account = synchronizer.getAccounts()[CURRENT_ZIP_32_ACCOUNT_INDEX]
```

Comment:
This is technically wrong; the accounts returned by `synchronizer.getAccounts()` happen to currently be in the order they were created and therefore for a single-seed wallet the first one happens to have ZIP 32 account index 0, but that's not something the API should be guaranteeing. But I note there's a TODO on this constant, so we can just fix this at the same time as that TODO is addressed.

Reply:
Yes, that was my intention. Besides that, the old Fragment-based Demo app will be removed.